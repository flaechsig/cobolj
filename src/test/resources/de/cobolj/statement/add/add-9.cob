IDENTIFICATION DIVISION.
PROGRAM-ID. HELLO.

DATA DIVISION.
   WORKING-STORAGE SECTION.
   01 WS-NUMBER-1 PIC 9(3) VALUES 50.
   01 WS-NUMBER-2 PIC 9(3)V9(2) VALUES 100.50.
   01 WS-NUMBER-RESULT PIC 9(1) VALUES 3.
   01 WS-NUMBER-RESULT2 PIC 9(3) VALUES 3.

PROCEDURE DIVISION.
   ADD 5.4 WS-NUMBER-1 TO WS-NUMBER-RESULT ROUNDED WS-NUMBER-RESULT2 ROUNDED
   ON SIZE ERROR
    DISPLAY 'SizeError'.
   DISPLAY WS-NUMBER-RESULT
   DISPLAY WS-NUMBER-RESULT2
STOP RUN.