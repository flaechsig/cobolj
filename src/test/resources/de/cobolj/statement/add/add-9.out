SizeError
3
058
